editor/editor_audio_buses.cpp

```diff
@@ -238,7 +238,14 @@ void EditorAudioBus::update_bus() {
 		fx->set_cell_mode(0, TreeItem::CELL_MODE_CHECK);
 		fx->set_editable(0, true);
 		fx->set_checked(0, AudioServer::get_singleton()->is_bus_effect_enabled(index, i));
-		fx->set_text(0, afx->get_name());
+		if (afx->get_name().is_empty()) {
+			fx->set_text(0, vformat("%s:%d", afx->get_class_name(), afx->get_instance_id()));
```

Comment:
You could maybe use `afx->to_string()` directly, which does pretty much the same (+ brackets).

Reply:
That being said in #42288 @Waranoi suggested using the filename instead.

I'm not sure it makes sense to use the instance ID as this will change between runs of the Godot process and I assume this name gets saved in the audio bus layout? It's not really useful information IMO, it's more the kind of stuff you see in the debugger.

So just the class name (possibly with some logic to prevent name collisions) or "Class (filename)" might be better.